hfysubs/inbox.py

```diff
@@ -46,6 +48,13 @@ def handle_inbox_stream():
 
 
 def construct_pm(author, subscriptions):
+    """
+    :type author: str
+    :type subscriptions: (str)[]
```

Comment:
I am not sure if `(type, type)` is the correct way to document tuples in python - but this makes sense to me, and was useful while verifying the method parameters.